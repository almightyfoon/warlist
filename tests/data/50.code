c35b1b_-0f6Z7teQ8f8tgcgc7Z7Z7S7T7W7-7-7Y7W7Y7Wgs__797w9vdi4i867H7I7H7I8M8I7-7-gs