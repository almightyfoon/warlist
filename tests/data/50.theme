{"allowed":[33,35,44,46,47,50,53,56,59,63,66,69,73,75,82,86,87,100,101,103,108,110,112,116,118,125,126,128,129,157,158,160,164,181,186,187,188,189,193,195,196,198,199,200,203,207,213,215,233,234,235,236,239,276,320,323,334,335,336,337,339,340,348,388,390,403,404,405,407,420,421,422,423,427,441,455,456,457,458,460,466,477,478,479,481,482,483,484,485,486,487,488,499,500,501,503,510,511,513,517,518,520,522,523,524,526,528,581,655,673,684,688,696,726,785,2066,3022,3026,3030,3033,3059,3060,3061,3062,3125,3155,3176,3177,3178,4014,4025,4026,4027,4034,4040,6005,6018,6030,6140,6160,6171,6187,6294,6302,6307,6308,6310,6328,6393,6396,6403,6404,6405,6406,6407,6408,6409,6410,6411,6412,6414,6420],"required":[128,129,130,132,136,137,139,151,152,153,155,157,160,164,180,187,188,198,199,200,207,210,658,2069,4011,6171,6308,6309,6396],"free":[129,152,160,181,183,185,186,187,188,189,193,195,196,198,200,207,210,211,658,821,3030,3142,4011,4027,6202,6303,6309,6311]}