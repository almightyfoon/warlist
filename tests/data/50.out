https://conflictchamber.com/#c35b1b_-0f6ZkTeQ8f8tgcgc7Z7Z7S7T7W7-7-7Y7W7Y7Wgs__797w9vdi4i867H7I7H7I8M8I7-7-gs

Khador Army - 75 / 75 points
[Theme] Winter Guard Kommand

[Butcher 1] Orsus Zoktavir, The Butcher of Khardov [+28]
 - Marauder [11]
 - Ruin [17]
 - War Dog [3]
Kovnik Jozef Grigorovich [4]
Winter Guard Artillery Kapitan [3]
Winter Guard Artillery Kapitan [3]
Winter Guard Field Gun Crew [0(4)]
Winter Guard Field Gun Crew [4]
Winter Guard Infantry (max) [10]
 - Winter Guard Infantry Officer & Standard [4]
 - Winter Guard Rocketeer (3) [6]
Winter Guard Mortar Crew [0(5)]
Winter Guard Mortar Crew [0(5)]
Winter Guard Rifle Corps (max) [13]
 - Winter Guard Rocketeer (3) [6]
Winter Guard Rifle Corps (max) [13]
 - Winter Guard Rocketeer (3) [6]
Bunker (Objective) [0]

Khador Army - 75 / 75 points

[Irusk 2] Supreme Kommandant Irusk [+27]
 - Behemoth [25]
 - Sylys Wyshnalyrr, the Seeker [4]
Iron Fang Kovnik [4]
Saxon Orrik [4]
Battle Mechaniks (min) [3]
Iron Fang Pikemen (max) [15]
 - Iron Fang Pikemen Officer & Standard [4]
Iron Fang Pikemen (max) [15]
 - Iron Fang Pikemen Officer & Standard [4]
Lady Aiyana & Master Holt [8]
 - Koldun Kapitan Valachev [4]
Winter Guard Mortar Crew [5]
Winter Guard Mortar Crew [5]
Bunker (Objective) [0]
